packages/camel-catalog/kaoto-camel-catalog-maven-plugin/src/test/java/io/kaoto/camelcatalog/CamelCatalogProcessorTest.java

```diff
@@ -200,15 +214,15 @@ public void testModelEnumParameter() throws Exception {
         checkEnumParameters(modelCatalog);
     }
 
-    @Test
+//    @Test
```

Comment:
This test is failing with 
```log
java.lang.NullPointerException: Cannot invoke "com.fasterxml.jackson.databind.JsonNode.asText()" because the return value of "com.fasterxml.jackson.databind.node.ObjectNode.get(String)" is null
```

![image](https://github.com/KaotoIO/kaoto-next/assets/16512618/2a85e59b-6f67-400e-8680-2931ffe3be39)

Reply:
Fixed the test